configs/MATEKF411WTE/config.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F411
+
+#define BOARD_NAME        MATEKF411WTE
+#define MANUFACTURER_ID   MTKS
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACC_SPI_BMI270
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_GYRO_SPI_BMI270
+#define USE_MAX7456
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_SOFTSERIAL
```

Comment:
As this is a build option we need add a guard to prevent duplicate defines.
```suggestion
#ifndef USE_SOFTSERIAL
#define USE_SOFTSERIAL
#endif
```